sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Serialization/SchemaRegistrySerializerOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Serialization;
+
+namespace Azure.Data.SchemaRegistry.Serialization
+{
+    /// <summary>
+    /// The options to use when configuring the <see cref="SchemaRegistrySerializer"/>.
+    /// </summary>
+    public class SchemaRegistrySerializerOptions
+    {
+        /// <summary>
+        /// Allows the user to pass in an <see cref="ObjectSerializer"/> with configured options.
+        /// The default is a <see cref="JsonObjectSerializer"/>.
+        /// </summary>
+        public ObjectSerializer Serializer { get; set; } = new JsonObjectSerializer();
```

Comment:
Does this need to be a unique `JsonObjectSerializer` instance?   Could we have a static one here that is shared by all options instances?